src/identity/hcs/did/hcs-did-message.ts

```diff
@@ -1,46 +1,41 @@
-import { PublicKey, Timestamp, TopicId } from "@hashgraph/sdk";
-import { Hashing } from "../../../utils/hashing";
+import { Timestamp, TopicId } from "@hashgraph/sdk";
+import Long from "long";
 import { TimestampUtils } from "../../../utils/timestamp-utils";
-import { DidDocumentBase } from "../../did-document-base";
-import { DidDocumentJsonProperties } from "../../did-document-json-properties";
 import { DidMethodOperation } from "../../did-method-operation";
-import { Decrypter, Encrypter, Message } from "../message";
-import { MessageEnvelope } from "../message-envelope";
+import { DidParser } from "../../did-parser";
+import { HcsDidEvent } from "./event/hcs-did-event";
+import { HcsDidEventParser } from "./event/hcs-did-event-parser";
 import { HcsDid } from "./hcs-did";
 
+export type Signer<T> = (message: T) => T;
+
 /**
  * The DID document message submitted to appnet's DID Topic.
  */
-export class HcsDidMessage extends Message {
+export class HcsDidMessage {
+    private static serialVersionUID = Long.fromInt(1);
+
+    protected timestamp: Timestamp;
     protected operation: DidMethodOperation;
     protected did: string;
-    protected didDocumentBase64: string;
-    /**
-     * The date when the DID was created and published.
-     * It is equal to consensus timestamp of the first creation message.
-     * This property is set by the listener and injected into the DID document upon calling getDidDocument() method.
-     */
-
-    protected created: Timestamp;
-    /**
-     * The date when the DID was updated and published.
-     * It is equal to consensus timestamp of the last valid update or delete message.
-     * This property is set by the listener and injected into the DID document upon calling getDidDocument() method.
-     */
-    protected updated: Timestamp;
+    protected event: HcsDidEvent;
 
     /**
      * Creates a new instance of {@link HcsDidMessage}.
      *
      * @param operation         The operation on DID document.
      * @param did               The DID string.
-     * @param didDocumentBase64 The Base64-encoded DID document.
+     * @param event             The DID Event.
      */
-    constructor(operation: DidMethodOperation, did: string, didDocumentBase64: string) {
-        super();
+    constructor(operation: DidMethodOperation, did: string, event: HcsDidEvent) {
```

Comment:
This appears to be a breaking change from the previous version (method parameters), is this acceptable ?

Reply:
Version 1.0 of the DID Method is a breaking change for 0.9 spec.